src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CheckpointsRule.cs

```diff
@@ -26,7 +26,7 @@ public override Task RunAsync(RuleContext context)
             }
 
             // Check whether to use checkpoint to skip block validation.
-            if (context.SkipValidation == false && this.Parent.ConsensusSettings.UseCheckpoints)
+            if ((context.SkipValidation == false) && (this.Parent.ConsensusSettings.UseCheckpoints))
```

Comment:
The two on the left – yes

The two on the right – no

Reply:
ah yeah, was on auto mode.